README.adoc

```diff
@@ -66,81 +65,51 @@ You need to have Docker installed. For installation instructions, refer to the o
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
-== Creating a web client using the default JAX-RS API
+== Creating a web client using the CXF reactive provider
```

Comment:
The example shown here is for the default JAX-RS API, not the CXF reactive provider, so we should undo this change.  The way to tell which reactive provider will be used is the `rx(...)` method.   If there are no parameters (as it is in this case), then it is the default JAX-RS Provider - and it will always return a `CompletionStage`. If it is a non-default provider, then there will be a class passed in to the `rx` method (like `rx(RxObservableInvoker.class)` for Jersey or `rx(ObservableRxInvoker.class)` for CXF).

Keep in mind that neither Jersey's nor CXF's provider ships with Liberty - both require that the user package the provider with their app.

Reply:
Ah, sorry, both. That was my interpretation of the notes you put in the other issue.